R/LScommon.R

```diff
@@ -16,13 +16,27 @@
 #
 
 # general functions
-.evaluatePriors       <- function(priors){
+.evaluatePriors       <- function(priors, type){
   for(p in 1:length(priors)){
     for(i in 1:length(priors[[p]])){
       tempP <- priors[[p]][[i]]
```

Comment:
so `tempP` isn't actually used here. But you could make the code look a lot simpler by just using `tempP`. I'd also call it `tempPI`, or `priorPI`, to make it more obvious that it's basically `priors[[p]][[i]]`. There's also a ton of `names(priors[[p]])[i]`. You could also extract this once and reuse that. 

Both comments are just style though, functionally this looks good to me!

Reply:
yeah, you are right, the `tempP` is not used, probably some disconnect between the floor and keyboard. I removed it and kept the `priors[[p]][[i]]` everywhere since the function returns a modified `priors` object so the changed things keep being changed.